utils/iio_common.c

```diff
@@ -257,12 +257,12 @@ struct iio_context * handle_common_opts(char * name, int argc,
 		int *err_code)
 {
 	struct iio_context *ctx = NULL;
+	struct iio_context_params params = { .timeout_ms = -1 };
```

Comment:
`-1` would mean infinite timeout (IIRC), keep it zero-initialized and Libiio will set it to the default timeout that corresponds to the backend used.

Reply:
Ah, I see, `-1` was used previously to prevent `iio_context_set_timeout()` from being called if a timeout wasn't specified at the command-line. Thanks for catching that copy/paste mistake!